content/blog/contributor-highlight-francesco-canovai/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Contributor Spotlight: Francesco Canovai"
+date: 2025-05-07T14:51:43+02:00
+draft: false
+image:
+    url: 
+    attribution:
+author: fdrees
+tags:
+ - testing
+ - github
+ - kubernetes
+ - postgresql
+ - open-source
+summary: "In a mini-series on this blog we highlight the work of the community.
+  Today we meet Francesco Canovai, Senior Software Development Engineer at EDB, 
+  and CloudNativePG maintainer." 
+---
+
+# Contributor Spotlight: 
+
+Building and maintaining and open source project takes a village. In a
+mini-series on this blog we would like to highlight the work of our
+maintainers, component owners, and members of the larger community.
+
+Today it's my great pleasure to introduce [Francesco Canovai](https://github.com/fcanovai/), 
+my colleage at EDB. Francesco is a Senior Software Development Engineer, 
+living in Prato, Italy. No point asking when he first found out about 
+CloudNativePG: he was in the room when/where the project started! The 
+first commit with his name on it is the initial suite of end-to-end tests.
+
+One dream he has for the project is to rework the tests to use a configuration 
+and avoid the GitHub Actions + Makefile + shell scripts + Ginkgo Frankenstein 
+mix currently used. If only there were unlimited hours in the day!
+
+Francesco got into technology in the first place playing videogames (of the 
+rpg and grand strategy genre). At 15 he was sure he wanted to study computer 
+science, and in his early twenties he really wanted to teach maths. It's 
+never too late to switch, I don't think the world is saturated for math 
```

Comment:
"I don't think" is a reflection from Floor, or from Francesco? Not clear IMO, and I think it would be good to make it more explicit.
My hunch is that it's Floor's.
Maybe could be converted to something like "It's never too late to switch, Francesco! I don't think the world ..."

Reply:
> he really wanted to teach maths

Good man. This is also something I think about.